lnst/Recipes/ENRT/TeamVsBondRecipe.py

```diff
@@ -1,5 +1,10 @@
-from lnst.Common.Parameters import IntParam, Param, StrParam
-from lnst.Common.IpAddress import ipaddress
+from lnst.Common.Parameters import (
+    Param,
+    StrParam,
```

Comment:
You removed `IntParam` which is used in the code later.

Reply:
Yeah i noticed that, fix incoming